### ISSUE TITLE
throwing Error if template literal with null placeholder

### ISSUE THREAD

**Issue by nor3** [skolmer/es2015-i18n-tag#65]
## current behavior

following code raise exception "Cannot read property 'toLocaleString' of undefined "

```
const mayNullOrUndefined = null;
i18n`${mayNullOrUndefined}`
```
 https://codesandbox.io/s/priceless-tdd-4nw3z

## expacted behavior

extract with "null" or "undefined" value for placeholder, which is normal template literal behavior.
It may work for detect bug, but unexpected in case the variable may take null/undefined value.

## workaround

```
i18n`${mayNullOrUndefined || ""}`
```
I hate this...

**Comment by skolmer** [comment]
sure, we all hate undefined ;) I will accept pull requests on this topic.

**Comment by skolmer** [comment]
thanks for your contribution.  change is released in [v1.6.1](https://github.com/skolmer/es2015-i18n-tag/releases/tag/v1.6.1)
